src/Controller/Report/OpenIssuesController.php

```diff
@@ -51,7 +51,7 @@ protected function defaultAction()
      */
     protected function prepareTemplate()
     {
-        $res = Report::getOpenIssuesByUser($this->prj_id, $this->cutoff_days ?: 7, $this->group_by_reporter);
+        $res = Report::getOpenIssuesByUser($this->prj_id, $this->cutoff_days, $this->group_by_reporter);
```

Comment:
if you removed the `7` here, where's the default set then? it's no longer `7`?

sorry, not familiar with this part of the codebase.